### PR TITLE
refactor: Commands respond to `Entrypoint()`

### PR DESCRIPTION
**Pull request by boblail**

### CODE REVIEW

File: options.go [hunk 1]
```diff
@@ -57,16 +57,16 @@ func PrependCommands(cmds ...interface{}) Option {
...
 
-func buildCommands(m Module, cmds []interface{}) []Command {
+func buildCommands(e *Entrypoint, cmds []interface{}) []Command {
 	var result []Command
 
 	for _, cmd := range cmds {
 		switch v := cmd.(type) {
 		case *EmbeddedCommand:
-			result = append(result, &builtinCommand{m, v})
+			result = append(result, &builtinCommand{e, e, v})
```
Comment (cmilesq) [inline options.go:66]: Both of these intentionally `e` ? `entrypoint` and `module` ?

Comment (cmilesq) [reply options.go:66]: Same with below I assume

File: builtin_command.go [hunk 2]
```diff
@@ -5,10 +5,12 @@ import (
 )
 
 type builtinCommand struct {
+	entrypoint *Entrypoint
 	parent     Module
 	definition *EmbeddedCommand
 }
 
+func (c *builtinCommand) Entrypoint() *Entrypoint  { return c.entrypoint }
```
Comment (cmilesq) [inline builtin_command.go:13]: Are we anticipating a consumer will need access to the internal object pointer from accessing it this way and that we won't be able to pass it explicitly to other consumers?

Comment (cmilesq) [reply builtin_command.go:13]: Similar exposure in other areas

**Review by cmilesq — APPROVED** [review]

**Comment by boblail** [comment]
Closed (for now) in favor of #6

**Pull request closed without merging**